cli/src/klio_cli/commands/job/utils/templates/MANIFEST.in.tpl

```diff
@@ -1 +1,2 @@
 include job-requirements.txt
+include klio-job-run-effective.yaml
```

Comment:
Are these template changes changes that we should make in our cookie cutters too? (i can definitely help with that when this is close to merging if so)